bin/configure-fastly.js

```diff
@@ -0,0 +1,176 @@
+                    async.apply(fastly.setFastlyHeader.bind(fastly), version.number, bucketNameHeader)
+                ], cb)
+            },
+            function (cb) {
+                async.series([
+                    async.apply(fastly.setCondition.bind(fastly), version.number, passCondition),
+                    function (cb) {
+                        async.parallel([
+                            async.apply(
+                                fastly.request.bind(fastly), 'PUT',
+                                fastly.getFastlyAPIPrefix(serviceId, version.number) + '/request_settings/Pass',
+                                {request_condition: PASS_REQUEST_CONDITION_NAME}
+                             ),
+                            async.apply(
+                                fastly.request.bind(fastly), 'PUT',
+                                fastly.getFastlyAPIPrefix(serviceId, version.number) + '/backend/femto',
+                                {request_condition: PASS_REQUEST_CONDITION_NAME}
+                            )
+                        ], cb)
+                    }
+                ], cb)
+            },
+            function (cb) {
+                async.series([
+                    async.apply(fastly.setCondition.bind(fastly), version.number, passCacheCondition),
+                    async.apply(
+                        fastly.request.bind(fastly), 'PUT',
+                        fastly.getFastlyAPIPrefix(serviceId, version.number) + '/cache_settings/Pass',
+                        {cache_condition: PASS_CACHE_CONDITION_NAME}
+                    ),
+                ], cb)
+            },
+            function (cb) {
+                async.forEachOf(routes, function (route, id, cb) {
+                    var condition = {
+                        name: getConditionNameForView(route.view),
+                        statement: 'req.url ~ "' + route.pattern + '"',
+                        type: 'REQUEST',
+                        priority: id
+                    };
+                    var header = {
+                        name: getHeaderNameForView(route.view),
+                        action: 'set',
+                        ignore_if_set: 0,
+                        type: 'request',
+                        dst: 'url',
+                        src: '"/' + route.view + '.html"',
+                        request_condition: getConditionNameForView(route.view),
+                        priority: 10
+                    };
+                    async.series([
+                        async.apply(fastly.setCondition.bind(fastly), version.number, condition),
+                        async.apply(fastly.setFastlyHeader.bind(fastly), version.number, header)
+                    ], cb);
+                }, cb)
+            }
+        ], next);
+    }], function (err, responses) {
+        if (err) {
+            console.log(err);
```

Comment:
If you want the process to exit with an error code (fatal exception), it might make more sense to throw an error so developers have a stack trace:
```js
throw new Error(err);
```